testing/simulated/pectra_withdrawal_test.go

```diff
@@ -541,9 +541,8 @@ func (s *PectraWithdrawalSuite) TestWithdrawalFromExcessStake_HasCorrectWithdraw
 	{
 		// Send Deposit Requests
 		iterations := int64(10)
-		s.defaultDepositWithNonce(blsSigner, creds, depositAmount, true, big.NewInt(0))
-		for i := 1; i < 30; i++ {
-			s.defaultDepositWithNonce(blsSigner, creds, depositAmount, false, big.NewInt(int64(i)))
```

Comment:
because this PR sets operator key in `SetDepositStorage`, we don't need to reset it again. If we do that, tx would revert and we'll have less deposit events than we expect